theano/gof/cmodule.py

```diff
@@ -269,7 +269,9 @@ def dlimport(fullpath, suffix=None):
     if not os.path.isabs(fullpath):
         raise ValueError('`fullpath` must be an absolute path', fullpath)
     if suffix is None:
-        if fullpath.endswith('.so'):
+        if fullpath.endswith('.pypy-41.so'):
```

Comment:
For this, why not use Python regex and if fullpath end with ".pypy*.so", we reuse that as the suffix? This won't be version dependant that way.